### DISCUSSION TITLE
best practices for serving fgb

### DISCUSSION THREAD

**Discussion by SpatialJ** [flatgeobuf/flatgeobuf#170] [General]
Hi,

are there any best practices on how to server fgb's? E.g. regarding caching, compression, http2, ...  

Best,
Johannes

**Comment by bjornharrtell** [comment]
Did you read https://flatgeobuf.org/#optimizing-remotely-hosted-flatgeobufs?

**Reply by SpatialJ** [reply]
yes, i did, but it it regarding cdn's. I was thinking more about an on premise use case using e.g. nginx or something similiar.

**Reply by bjornharrtell** [reply]
A CDN is just a glorified caching webserver in my opinion. :) So same principles apply, unless I'm mistaken.

**Reply by bjornharrtell** [reply]
Ok a CDN is distributed but the same principles still apply.
